autodocs/README.md

```diff
@@ -0,0 +1,57 @@
+## Autodocs Maintenance Tools
+Tools for documentation maintenance. 
+
+## Installation
+
+```shell
+cd autodocs
+composer install
+```
+## Usage
+
+### Audit Summary
+
+```shell
+./autodocs audit
+```
+
+### Get deprecated articles
+
+```shell
+./autodocs audit --deprecated
+```
+
+### Get articles with invalid timestamps
+
+```shell
+./autodocs audit --invalid
+```
+
+### Get top articles from analytics csv
+
+```shell
+./autodocs audit topcontent data=input.csv
```

Comment:
Not a blocking comment, just a question - will we have an "easy" way of getting this data?

Reply:
yes! I am working on a spreadsheet with the results from this, for our first sprint it will be more manual but the plan is to automate as much as possible later on.